plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -287,4 +314,96 @@ public void dropSchema(ConnectorSession session, String schemaName)
+                                .outputType(type)
+                                .build()));
+
+        numericTypes.forEach(type ->
+                builder.add(
+                        SingleInputAggregateFunction.builder()
+                                .prestoName("sum")
+                                .inputTypes(type)
+                                .outputType(type)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("avg")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("avg(CAST(%s AS double))")
+                                .jdbcTypeHandle(doubleHandle)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("stddev")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("CASE WHEN COUNT(%s) > 1 THEN STDDEV(%s) ELSE NULL END")
+                                .jdbcTypeHandle(doubleHandle)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("stddev_pop")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("CASE WHEN COUNT(%s) = 0 THEN NULL WHEN COUNT(%s) = 1 THEN 0.0 ELSE STDDEV_POP(%s) END")
+                                .jdbcTypeHandle(doubleHandle)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("stddev_samp")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("CASE WHEN COUNT(%s) > 1 THEN STDDEV_SAMP(%s) ELSE NULL END")
+                                .jdbcTypeHandle(doubleHandle)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("variance")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("CASE WHEN COUNT(%s) > 1 THEN VARIANCE(%s) ELSE NULL END")
+                                .jdbcTypeHandle(doubleHandle)
+                                .build())
+                        .add(SingleInputAggregateFunction.builder()
+                                .prestoName("var_pop")
+                                .inputTypes(type)
+                                .outputType(DOUBLE)
+                                .expression("CASE WHEN COUNT(%s) = 0 THEN NULL WHEN COUNT(%s) = 1 THEN 0.0 ELSE VAR_POP(%s) END")
```

Comment:
Druid's `COUNT(x)` has different semantics depending on if `null`s are configured in Druid.
In absence of `null`s Druid considers the zero-value of each type (e.g. empty string for string, 0 for long etc.) as `NULL` and they get ignored in functions like `COUNT(x)`.

Won't this be a correctness issue to use `COUNT` in pushdown expressions?